src/main/webapp/app/shared/util/core-drive-annotation-submission/core-drive-annotation-submission.ts

```diff
@@ -0,0 +1,136 @@
+import _ from 'lodash';
+import { Drug, Gene, Mutation, Review, Treatment, Tumor, Vus, DrugCollection } from '../../model/firebase/firebase.model';
+
+export function getGeneData(geneData: Gene, excludeComments: boolean, onlyReviewedContent: boolean, drugList: DrugCollection): true | Gene {
```

Comment:
Should I review this file? I assume you refactored the code significantly?

Reply:
Any inside of `src/main/webapp/app/shared/util/core-drive-annotation-submission` is new code. Anything in `src/main/webapp/app/shared/util/core-evidence-submission` is old code, but was move to that folder for organizational purposes.